website/docs/maintenance/tiered-storage/lakehouse-storage.md

```diff
@@ -22,7 +22,9 @@ Lakehouse Storage is disabled by default, you must enable it manually.
 
 ### Lakehouse Storage Cluster Configurations
 #### Modify `server.yaml`
```

Comment:
TBH, I think mix paimon and iceberg in this page make it harder for users to focus on the datalake that they care about? Could we move these steps about iceberg to `integrate-data-lakes/iceberg.md` directly? It will be much clear. Let's use just consider this page as an example for paimon only.